content/en/docs/guides/compression.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Compression
+description: >-
+    How to use compression to reduce the bandwidth used by gRPC
+---
+
+### Overview
+
+Compression is used to reduce the amount of bandwidth used between peers. The compression supported by gRPC acts at the individual message level.
+
+Depends on language, it's possible to control compression settings at channel, call and message level. Different language also support different compression algorithms including customized compressor. 
+
+### Compression Method Asymmetry Between Peers
+
+gRPC allows asymmetric compression communication, whereby a response may be compressed differently, if at all. A gRPC peer may choose to respond using a different compression method to that of the request, including not performing any compression, regardless of channel and RPC settings (for example, if compression would result in small or negative gains).
```

Comment:
"may be compressed differently"

differently from what? From the request or each response is different?